paper-submenu.html

```diff
@@ -131,9 +131,9 @@
        * Expand the submenu content.
        */
       open: function() {
-        if (!this.disabled && !this._active) {
+        if (!this.disabled && !this.opened) {
           this.$.collapse.show();
-          this._active = true;
+          this.opened = true;
           this.__trigger && this.__trigger.classList.add('iron-selected');
```

Comment:
What do you think about moving this into `_openedChanged`? Since `opened` isn't 'private' and this class seems to be intended to mirror its state, that seems like the right spot to update it now.